src/screens/level.py

```diff
@@ -284,6 +288,11 @@ def interact(self):
                 self.start_day_transition()
             if collided_interactions[0].name == "Trader":
                 self.switch_screen(GameState.SHOP)
+            if collided_interactions[0] in self.bush_sprites.sprites():
+                if self.player.axe_hitbox.colliderect(
+                    collided_interactions[0].hitbox_rect
+                ):
+                    collided_interactions[0].hit(self.player)
```

Comment:
Using the axe to get berries from a bush feels a bit like overkill to me. It's as if you were using a bazooka to kill a mosquito 🤣

Reply:
I'm not specifically using the axe to harvest, that would indeed be overkill^^ I'm just using the hitbox of the axe to determine if the bush is in range, as the player hitbox alone is too big for that. Berries are harvested by simply clicking

Reply:
Ah, phew. You got me worried 🤣